server/components/form-settings/_form-settings.scss

```diff
@@ -86,6 +86,10 @@
   align-self: center !important;
 }
 
+.form__settings-hide {
```

Comment:
If you want to do it the `BEM` way. It should be like: `.form__settings--is-hidden`. Though, you can also use the utility class `u--is-hidden`, which does the same.